client/public/static/js/account/update_recipe.js

```diff
@@ -4,7 +4,8 @@ $(document).ready(function () {
     const username = utils.getUserNameFromCookie();
     const userId = await utils.getUserIdFromUsername(username);
     let formRecipeName = document.getElementById('recipeName');
-    
+  
+    //delete currently existing recipe as we will replace it with a "new" recipe
```

Comment:
Before being able to submit, we should check if any of the data actually changed (see my profile or my dietary for what I mean). Prevent the user from updating if nothing has changed from the original load. 

Also why delete/create and not update? Question because on the refactor db branch we'd need to make multiple calls to the database:
- one to delete the recipe
- one to insert a new record
- one to delete the recipe reference from the users favorites
- one to insert the new object id

If we just update the recipe, we only make one trip to the database and then we can use the recipes collection more openly especially for publishing/displaying. Let me know your thoughts!

Reply:
Refactored this all to now be an update as it should have been instead of delete and create